### PR TITLE
Fix the DEBUG build for QEMU_V8

### PR DESCRIPTION
**Pull request by Hmtir**
When the DEBUG config was set, the optee_os was still built with
DEBUG=0 because the flag was hardcoded. As a result the optimization
was still O2. This is fixed by using the global DEBUG flag instead of
the hardcoded value of zero.

Signed-off-by: Hashem Tatari <hashem.tatari@exset.com>

### CODE REVIEW

File: qemu_v8.mk [hunk 1]
```diff
@@ -133,7 +133,7 @@ linux-cleaner: linux-cleaner-common
 # OP-TEE
 ################################################################################
 OPTEE_OS_COMMON_FLAGS += PLATFORM=vexpress-qemu_armv8a CFG_ARM64_core=y \
-			 DEBUG=0 CFG_PM_DEBUG=0
+			 DEBUG=$(DEBUG) CFG_PM_DEBUG=0
```
Comment (etienne-lms) [inline qemu_v8.mk]: by the way, `CFG_PM_DEBUG` is a deprecated thing, should be removed.

Comment (Hmtir) [reply qemu_v8.mk]: Yes I was not sure about that that's why I didn't change it.
I will also remove it then.

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>` with jforissier's comment addressed.
@Hmtir, can you update the P-R? It looks fine and is valuable.

**Comment by jbech-linaro** [comment]
Closing the PR, since I've manually merged the patches (after fixing Jerome's comment and adding the tags).

**Pull request closed without merging**